third_party/move/move-compiler/tests/move_check/parser/aptos_stdlib_attributes.exp

```diff
@@ -6,6 +18,18 @@ error[E02001]: duplicate declaration, item, or annotation
   │       │   
   │       Attribute previously given here
 
+warning[W02016]: unknown attribute
+  ┌─ tests/move_check/parser/aptos_stdlib_attributes.move:7:7
+  │
+7 │     #[testonly]
+  │       ^^^^^^^^ Attribute name 'testonly' is unknown (use --skip-attribute-checks CLI option to ignore); known attributes are '{"bytecode_instruction", "deprecated", "expected_failure", "native_interface", "test", "test_only", "verify_only"}'.
```

Comment:
Not a comment for this PR, but this seems like a good candidate for the move linter (to identify the closest allowed attribute and suggest it as fix).

Reply:
Well, there is a list displayed.  Good point, though.